### DISCUSSION TITLE
Eigene icons benutzen / How to use your own icons

### DISCUSSION THREAD

**Discussion by Zefau** [Zefau/ioBroker.jarvis#1297] [Q&A]
Um eigene Icons zu benutzen ist die Grafik mittels Base64 Image Encoder (https://www.base64-image.de/) in eine Zeichenkette umzuwandeln. Diese beginnt nach dem Konvertieren mit `data:image/jpeg;base64` (oder `data:image/png;base64`).

Diese Zeichenkette in das Icon Input-Feld einfügen.

_____

In order to use your own icons you need to convert the image to a base64 string using Base64 Image Encoder (https://www.base64-image.de/). After the conversion the string starts with `data:image/jpeg;base64` (or `data:image/png;base64`).

Paste the code into the icon input field.

**Comment by Minta79** [comment]
"Diese Zeichenkette in das Icon Input-Feld einfügen."
Welches Feld ist mit icon Input Feld gemeint?

**Comment by mcuiobroker** [comment]
https://mcuiobroker.gitbook.io/jarvis-infos/jarvis/beispiele/geraete-einstellungen/icons-eigene/eigene-icons
